style.css

```diff
@@ -0,0 +1,240 @@
+body {
+  margin-right: 100px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 20px;
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 800;
+}
+
+.newsletter {
+  color: #00bcfc;
+  font-weight: 900;
+  font-family: "PT SERIF", Georgia, "Times New Roman", Times, serif;
+  letter-spacing: -1px;
+}
+
+.newsletter:hover {
+  color: #89cff0;
+}
+
+.s {
+  color: #999;
+}
+
+.s:hover {
+  color: #999;
+  text-decoration: underline;
+}
+
+.signup {
+  margin-left: 90px;
+  display: flex;
+  flex-direction: column;
+  margin-right: 50px;
+  font-family: "PT Serif", Georgia, sans-serif;
+}
+
+.newscientist {
+  font-size: 60px;
+  letter-spacing: -5px;
+  font-family: "PT SERIF", Georgia, "Times New Roman", Times, serif;
+  margin-right: 50px;
+}
+
+.halfpricesale {
+  color: white;
+  background-color: #00bcfc;
+  padding: 10px;
+  font-family: "PT SERIF", Georgia, "Times New Roman", Times, serif;
+  font-weight: 900;
+}
+
+.sticky {
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  color: white;
+  padding-top: 1px;
+  background-color: white;
+}
+
+.navlinks {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  margin-left: 30px;
+}
+
+.more {
+  list-style: none;
+}
+
+.b {
+  font-weight: 800;
+  color: black;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  width: 10rem;
+  padding-bottom: 20px;
+  padding-right: 20px;
+  z-index: 1;
+  background-color: #f9f9f9;
+}
+
+.ext-more:hover .dropdown-content {
+  display: block;
+}
+
+.nav-link:hover {
+  color: #00bcfc;
+  text-decoration: #000 underline 5px;
+  text-underline-offset: 5px;
+}
+
+.n {
+  color: #00bcfc;
+}
+
+.headerline {
+  height: 1px;
+  width: 200%;
```

Comment:
- [ ] There’s an overflow on your page as a result of the width added to the class. You can fix this by removing the width .